### DISCUSSION TITLE
Need extracted values in key-value like.

### DISCUSSION THREAD

**Discussion by ananan** [projectdiscovery/nuclei#2821] [Q&A] [answered]
Hi Team,

by now the nuclei can return matched extracted values in a list, can it return in key-value like ?

if just return values list, we don't know which values means for. 

for example, if I want to extracting web title and server from response, it return 2 items in a disorder list, I don't know which is the value i want.

thanks

**Comment by forgedhallpass** [comment]
Hello @ananan,

Could you please give an example?

**Reply by ananan** [reply]
Sure, for example, the below image is the output of cmd:
`nuclei -t web-info-collector.yaml -ni -duc  -u http://xiangsite.com/ -json -nm -ms -o res.json|jq`
and my rule:
<img width="303" alt="image" src="https://user-images.githubusercontent.com/12604711/200854939-a46f1f2a-1a5b-4911-9bac-5b0ebcd3649e.png">


take a look at the extracted-results field, I can't figure it out which is **title** field and **server** field
<img width="848" alt="image" src="https://user-images.githubusercontent.com/12604711/200853070-9459138c-73e7-472e-a6f6-0b00c89c35bd.png">

**Reply by olearycrew** [reply, marked as answer]
@ananan take a look at Katana field selection: https://nux.gg/katana. I think that has more of what you're looking for here!
